wnp-server/src/main/java/com/wonnapark/wnpserver/global/auth/JwtAuthorizationFilter.java

```diff
@@ -75,25 +75,17 @@ private String extractTokenFromHeader(HttpServletRequest request, String headerN
         return null;
     }
 
-    private boolean validateRefreshToken(String refreshToken, Long userId) {
-        if (authenticationResolver.isValidToken(refreshToken)) {
-            RefreshTokenResponse refreshTokenResponse = authenticationResolver.checkExpiredRefreshToken(userId);
-            if (refreshTokenResponse.refreshToken().equals(refreshToken))
-                return true;
-        }
-        return false;
-    }
-
-    private void setErrorResponse(HttpServletResponse response, ErrorCode errorCode, String redirectUri) throws IOException {
+    private void setErrorResponse(HttpServletResponse response, JwtInvalidException exception, String redirectUri) throws IOException {
+        ErrorCode errorCode = exception.getErrorCode();
         response.setStatus(errorCode.getValue());
         response.setContentType("application/json");
         response.setCharacterEncoding("UTF-8");
-//        response.sendRedirect(redirectUri);
-//        프론트가 없어서 백엔드로 다시 요청이 계속 돌아오기 때문에 주석 처리함
+        response.setHeader(HttpHeaders.LOCATION, redirectUri);
         ErrorResponse errorResponse = ErrorResponse.create(errorCode);
         try {
             String json = objectMapper.writeValueAsString(errorResponse);
             response.getWriter().write(json);
+            log.warn("터진 곳 -> {}", exception.getStackTrace());
```

Comment:
이거 뒤에 e 를 달면 stacktrace 가능한 걸로 알고 있습니다.
```suggestion
            log.warn("토큰 예외 정보 : {}", errorResponse, e);
```